.github/workflows/export.yml

```diff
@@ -17,18 +18,25 @@ jobs:
       AWS_S3_ENDPOINT: ${{ secrets.DO_S3_ENDPOINT }}
       AWS_ACCESS_KEY_ID: ${{ secrets.DO_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
-      AWS_S3_BUCKET: edm-recipes
+      AWS_S3_LIBRARY_BUCKET: edm-recipes
     strategy:
       fail-fast: false
+      max-parallel: 1
       matrix:
         entity:
-          - dcp_projects
```

Comment:
They are live under `include` now:
```
include:
          - entity: 'dcp_projects'
            open: true
          - entity: 'dcp_projectbbls'
            open: true
```
This is the way to run them with the `open` variable set equal to true so that the new export process runs for just these two datasets